lib/es5-helpers.js

```diff
@@ -9,35 +9,27 @@ let prefixPath = null
 const parentPath = FS.realpathSync(Path.join(__dirname, '..'))
 const atomEslintPath = Path.join(parentPath, 'node_modules', 'eslint')
 
+// Find directory of eslint installation. Traverse up the tree checking every
+// `node_modules` directory
 function findEslintDir(params) {
-  const modulesPath = find(params.fileDir, 'node_modules')
-  let eslintNewPath = null
+  if (!params.global) {
+    return find(params.fileDir, 'node_modules/eslint') || atomEslintPath
```

Comment:
It seemd to work when I was debugging. with `eslint` it won't go inside `node_modules`, would it?